bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -1105,6 +1139,9 @@ void ViewsManager::processComplaintsFromViewChangeMessage(ViewChangeMsg* msg,
   MsgSize size = 0;
   int numberOfProcessedComplaints = 0;
 
+  // Complaints for higher view should aim to reach the view of the message.
+  targetView = msg->newView();
```

Comment:
I think it might be more readable if we replace **msg->newView()** with **targetView** in the references inside the function **processComplaintsFromViewChangeMessage**. This way we show that this is the View we are checking if we have sufficient evidence we need to reach.